notifyQueue.js

```diff
@@ -32,6 +32,22 @@ if (process.env.NODE_ENV === 'prod') {
 }
 
 const notifyQueue = new Queue('sendMail', redisConf);
-setQueues([notifyQueue]);
 
-module.exports.UI = UI;
+const serverAdapter = new ExpressAdapter();
+
+createBullBoard({
+  queues: [
+    new BullAdapter( notifyQueue ),
+  ],
+  serverAdapter 
+})
+
+function getRouter( basePath ) {
+	if ( basePath ) {
```

Comment:
Just for clarity, do we still want to return getRouter when basePath is not present or what will happen if there is no basepath

Reply:
Yes we want to return it because the function "getRouter" is the function that can handle the express server request.

It is just I didn't wanted to set a basePath when there is none. But after looking at my code that call it, we would always have a basePath. I will remove the conditional.